### ISSUE TITLE
how to change directions?

### ISSUE THREAD

**Issue by mspanish** [pixijs/pixi-particles#46]
Sorry I must be missing something really obvious, but how do I change a burst or stream from 1 direction to another? For example the bubble spray example - how would I simply reverse this to spray downwards instead of up?

While the samples are all lovely, lack of basic controls like changing direction make this editor very hard to actually use.

**Comment by andrewstart** [comment]
Initial direction is the Start Rotation - 0 is to the right, and 90 is down, values are 0-360.

**Comment by mspanish** [comment]
Thanks, sorry to have asked something so obvious - I was pretty sure I tried that one but maybe not.

**Comment by andrewstart** [comment]
The emission arc/direction created by the start rotation affects image rotation, direction of movement, and the path traveled by a PathParticle. Additionally, Emitters have a `rotate()` method to rotate an already created emitter.

**Comment by mspanish** [comment]
Thanks Andrew, this is really great work. The only other thing I'd beg for is if you guys have any more pre-done JSON samples to put into the editor - the more the merrier! I'm happy to submit ones I've altered as well - as I'm super new to particles, adapting the emitters to do exactly what I want has proven a bit difficult for me. I'm doing a chlidren's app and mostly need things like happy bursts of stars and such - and the samples seem more focused on things people would need for gaming apps (with good reason, I'm sure that's what most people are making).  The bubbles are great and I was able to adjust them quite easily to make a nice slow stream of bubbles for an undersea scene.

Thanks again for all you do, I know open source work doesn't exactly pay one in material rewards!
